presto-product-tests/src/main/java/com/facebook/presto/tests/cli/PrestoCliTests.java

```diff
@@ -123,14 +152,35 @@ public void shouldRunQueryFromFile()
         temporayFile.deleteOnExit();
         Files.write("select * from hive.default.nation;\n", temporayFile, UTF_8);
 
-        launchPrestoCliWithServerArgument("--file", temporayFile.getAbsolutePath());
+        launchPrestoCliWithServerSpecified("--file", temporayFile.getAbsolutePath());
         assertThat(trimLines(presto.readRemainingOutputLines())).containsAll(nationTableBatchLines);
     }
 
-    private void launchPrestoCliWithServerArgument(String... arguments)
+    private void launchPrestoCliWithServerSpecified(String... arguments)
```

Comment:
Not convinced. New name seems even more confusing to me.
Whatever. No strong opinion here.